lte/gateway/c/oai/lib/3gpp/3gpp_24.301.h

```diff
@@ -180,7 +180,9 @@ typedef struct ue_network_capability_s {
 #define UE_NETWORK_CAPABILITY_UIA7 0b00000001
   uint8_t uia : 7;
   /* Bits 8 to 6 of octet 7 are spare and shall be coded as zero */
```

Comment:
This comment seems outdated since you replaced the spare bits with specific ones here

Reply:
Removed